reporthandling/attacktrack/v1alpha1/attacktrackmethods.go

```diff
@@ -57,6 +57,25 @@ func (at *AttackTrack) Iterator() IAttackTrackIterator {
 	}
 }
 
+// GetSubstepsWithVulnerabilities returns a list of substeps names that check for vulnerabilities
+func (at *AttackTrack) GetSubstepsWithVulnerabilities() []string {
+	var substepNames []string
+
+	var traverse func(step AttackTrackStep)
+	traverse = func(step AttackTrackStep) {
+		if step.ChecksVulnerabilities {
```

Comment:
You can use the DoesCheckVulnerabilities() method

Reply:
@YiscahLevySilas1 right, fixed